### ISSUE TITLE
Pagination changes in V2?

### ISSUE THREAD

**Issue by hughlawt4** [diegomura/react-pdf#1047]
Apologies if this is not the place for general questions / request for help...

V2 fixing a lot of performance issues for me, but one thing I can't seem to get working is outputting pagination numbers on a page, I previously used a `<Text>` component with a `render` prop, but using the same doesn't seem to output anything in the V2 branch. Any pointers on where to look or if the implementation is different in this version?

Previously used:
```
<Text render={({ pageNumber, totalPages }) => `${pageNumber} / ${totalPages}`} />
```

I'm generating my PDF using the following if that gives any more insight:

```
const doc = session => (
  <Document>
    <Page size="A4" style={styles.titlePage} wrap>
      <View>
        <Text style={styles.title}>{courseName}</Text>
        <Text style={styles.subTitle}>{examname}</Text>
        <Text style={styles.subTitle}>Candidate: {session.index}</Text>
        <Text style={styles.subTitle}>
          Date taken: {moment(session.dt_started).format('DD/MM/yyyy')}
        </Text>
      </View>
      <Image
        src={`${__dirname}/image.png`}
        allowDangerousPaths
        style={styles.beImageLarge}
      />
    </Page>
    {session.responses.map((response, i) => (
      <Page size="A4" style={styles.page} key={`page-${i}`}>
        <View style={styles.header} fixed>
          <Text>{examname}</Text>
          <Text>Candidate: {session.index}</Text>
        </View>
        <View key={i} style={styles.section}>
          <View style={styles.question}>
            {processMarkup(getQuestionText(response.item_reference))}
          </View>
          <View style={styles.answer}>
            <Text style={styles.answerText}>STUDENT ANSWER</Text>
            {response.attempted ? (
              processMarkup(response.response, true)
            ) : (
              <Text style={styles.note}>Not attempted</Text>
            )}
          </View>
        </View>
        <View style={styles.footer} fixed>
          <Text render={({ pageNumber, totalPages }) => `${pageNumber} / ${totalPages}`} />
          <Image
            src={`${__dirname}/image.png`}
            allowDangerousPaths
            style={styles.beImage}
          />
        </View>
      </Page>
    ))}
  </Document>
);
```

Thanks for any and all help on this.

**Comment by rykon** [comment]
Could it be related to [issue#931](https://github.com/diegomura/react-pdf/issues/931)?

**Comment by ddcech** [comment]
When i add the `fixed` prop to the Text component, it shows the page numbers. For example like this
```
      <Text
          render={({ pageNumber, totalPages }) => `${pageNumber} / ${totalPages}`}
          fixed
      />
```

**Comment by diegomura** [comment]
Pagination numbers should be fixed in the last 2.0 versions! You can see it working in https://v2.react-pdf.org/repl .Sorry for the delay. Closing this for the moment. Please let me know if still happens.

**Comment by jan-rycko** [comment]
Problem still occurs for me in version 2.0.15. Tried everything:
- fixed Text component at the bottom of Page component
- fixed and non-fixed Text component inside fixed View component
- positioned absolutely
- in flexbox document flow
- adding fixed width and height
Render function seems to be fired twice as documentation stated. But the result is not used I guess. Wrapper rendered is empty: https://gyazo.com/ef13961011008472411d5c1f3623b755
It would be of size 0x0 if not for applied style (which was my first guess of why it's not showing up):
`<Text
          fixed
          debug
          style={[{
            position: "absolute",
            color: 'black',
            display: 'flex',
            fontSize: '12px',
            opacity: 1,
            bottom: 20,
            right: 20,
            left: 20,
            textAlign: "center",
            height: '12px',
            minHeight: '12px',
          }]}
          render={({ pageNumber, totalPages }) => {
            console.log('RENDER', pageNumber, totalPages)
            return (pageNumber + " / " + totalPages);
          }}
        />`
